node/coinstacks/gnosis/pulumi/index.ts

```diff
@@ -0,0 +1,95 @@
+import { readFileSync } from 'fs'
+import { deployCoinstack } from '../../../../pulumi/src/coinstack'
+import { Outputs, CoinServiceArgs, getConfig } from '../../../../pulumi/src'
+
+//https://www.pulumi.com/docs/intro/languages/javascript/#entrypoint
+export = async (): Promise<Outputs> => {
+  const appName = 'unchained'
+  const coinstack = 'gnosis'
+  const sampleEnv = readFileSync('../sample.env')
+  const { kubeconfig, config, namespace } = await getConfig()
+
+  const coinServiceArgs = config.statefulService?.services?.map((service): CoinServiceArgs => {
+    switch (service.name) {
+      case 'daemon':
+        return {
+          ...service,
+          ports: {
+            'daemon-http': { port: 8545 },
+            'daemon-ws': { port: 8546 },
+            'daemon-beacon': { port: 8551, ingressRoute: false },
+          },
+          configMapData: {
+            'jwt.hex': readFileSync('../daemon/jwt.hex').toString(),
+          },
+          readinessEndpoint: '/health',
```

Comment:
wish more nodes exposed this :)

Reply:
Given how fast nethermind is, might be moving all geth nodes to it down the line

Reply:
certainly new integrations if supported. once geth is synced, it is pretty bullet proof. would be interesting to see if rpc request performance differs a lot between the two. that would be a good reason to upgrade for sure